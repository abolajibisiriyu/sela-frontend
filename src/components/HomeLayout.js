import React from "react";
import Layout from "../blocks/Layout";
import AppFooter from "./Footer";

function HomeLayout(props) {
  return (
    <Layout>
      {props.children}
      <AppFooter />
    </Layout>
  );
}

export default HomeLayout;
