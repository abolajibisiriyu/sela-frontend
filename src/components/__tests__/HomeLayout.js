import React from "react";
import renderer from "react-test-renderer";

import HomeLayout from "../HomeLayout";

describe("HomeLayout Component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<HomeLayout />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
