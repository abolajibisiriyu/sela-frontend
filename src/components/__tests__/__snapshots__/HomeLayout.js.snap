// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLayout Component renders correctly 1`] = `
<div
  className="sc-bxivhb dvXZAB"
>
  <div
    className="sc-ifAKCX dihmFK"
  >
    (c) Sela. 
    2019
  </div>
</div>
`;
